sdk/python/kfp/_client.py

```diff
@@ -81,7 +81,7 @@ class Client(object):
   IN_CLUSTER_DNS_NAME = 'ml-pipeline.{}.svc.cluster.local:8888'
   KUBE_PROXY_PATH = 'api/v1/namespaces/{}/services/ml-pipeline:http/proxy/'
 
-  def __init__(self, host=None, client_id=None, namespace='kubeflow'):
+  def __init__(self, host=None, client_id=None, namespace='kubeflow', other_client_id=None, other_client_secret=None):
```

Comment:
can I suggest calling these new parameters `iap_client_id` and `iap_client_secret`? I think `other` is ambiguous as to what they is used for.

Reply:
They are in fact other_client_id not the iap_client_id. 
The first client_id is the iap_client_id; the second client_id/client_secret is non-IAP but used to fetch the refresh_tokens, which is later used to fetch id_tokens for the IAP endpoint.
Check out the https://cloud.google.com/iap/docs/authentication-howto#authenticating_from_a_desktop_app for more information.

Reply:
I see, thanks for clarifying.